osgp/protocol-adapter-dlms/osgp-dlms/src/main/java/org/opensmartgridplatform/dlms/services/ObjectConfigService.java

```diff
@@ -27,16 +27,19 @@
 @Slf4j
 public class ObjectConfigService {
 
-  private List<DlmsProfile> dlmsProfiles;
+  private final List<DlmsProfile> dlmsProfiles = new ArrayList<>();
```

Comment:
Deze zou ik initieel niet vullen met een lege lijst, maar in de constructor toekennen

Reply:
Yes, ga ik aanpassen.